cmd/vcluster/cmd/start.go

```diff
@@ -66,31 +71,42 @@ func ExecuteStart(ctx context.Context, options *options.VirtualClusterOptions) e
 	}
 
 	// get current namespace
-	currentNamespace, err := clienthelper.CurrentNamespace()
+	controlPlaneConfig, controlPlaneNamespace, controlPlaneService, workloadConfig, workloadNamespace, workloadService, err := pro.GetRemoteClient(options)
 	if err != nil {
 		return err
 	}
+	options.ServiceName = workloadService
+	_ = os.Setenv("NAMESPACE", workloadNamespace)
```

Comment:
Shouldn't we handle/exit if this error is non-nil?

Reply:
Can add that yes